core/src/main/java/com/datastrato/gravitino/tag/TagManager.java

```diff
@@ -61,6 +71,16 @@ public TagManager(IdGenerator idGenerator, EntityStore entityStore) {
       throw new RuntimeException(errorMsg);
     }
 
+    if (!(entityStore instanceof SupportsExtraOperations)) {
```

Comment:
From my current understanding, the addition of `SupportsExtraOperations` in this PR means that these may be an optional feature? If so, is it appropriate to throw an exception here?

Reply:
It is required for Tag, not optional, so that's why I throw an exception here if it is not supported.